sdk/appconfiguration/azure-appconfiguration-provider/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+* Updated to use AZURE_APP_CONFIGURATION_TRACING_DISABLED environment variable to disable tracing.
+* Changed default max retry to 2 from 3.
+* Changed default retry backoff max to 1 from 2.
```

Comment:
Just by looking at these changelogs, I am not sure what the difference is between "max retry" and "retry backoff max". Can you briefly explain the behavior we are changing?
For example: 
"Changed the maximum number of retries to 2 from the default of 3 retries."
"Changed the maximum back off time between retries to 1 minute from the default of 2 minutes."